### PR TITLE
[ISSUE #202]format codes and correct comments for nacos-api

### PR DESCRIPTION
**Pull request by universefeeler**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

for slove issue #202 

## Brief changelog

format some codes and correct some spell and adjust a little structure

## Verifying this change

please check

### CODE REVIEW

File: api/src/main/java/com/alibaba/nacos/api/config/filter/IConfigResponse.java [hunk 1]
```diff
@@ -17,25 +17,23 @@
...
+    Object getParameter(String key);
 
-	/**
-	 * get context
-	 * 
-	 * @return configContext
-	 */
-	public IConfigContext getConfigContext();
+    /**
+     * get config context
```
Comment (chenlushun) [inline api/src/main/java/com/alibaba/nacos/api/config/filter/IConfigResponse.java]: Capitalize the first letter

File: api/src/main/java/com/alibaba/nacos/api/naming/NamingService.java [hunk 2]
```diff
@@ -24,12 +24,14 @@
...
 /**
+ * Naming Service
+ *
  * @author dungu.zpf
  */
 public interface NamingService {
 
     /**
-     * Register a instance to service
+     * register a instance to service
```
Comment (chenlushun) [inline api/src/main/java/com/alibaba/nacos/api/naming/NamingService.java]: Why should the first letter be changed to lowercase?

File: api/src/main/java/com/alibaba/nacos/api/config/filter/IConfigResponse.java (hunk 1 above)
Comment (universefeeler) [reply api/src/main/java/com/alibaba/nacos/api/config/filter/IConfigResponse.java]: if necessary, i will make a unified revision

**Comment by nkorange** [comment]
Please resolve the conflicts.

**Pull request merged**